integration_tests/__tests__/snapshot-test.js

```diff
@@ -71,6 +72,11 @@ describe('Snapshot', () => {
     expect(
       content['snapshot is not influenced by previous counter 0']
     ).not.toBe(undefined);
+
+    const info = result.stderr.toString();
+    expect(info.includes('4 snapshots written in 2 test files')).toBe(OK);
```

Comment:
You're superright! 
I've always seen `toMatch` used with regular expressions so I didn't though about using it with the string version, which is still used to construct a regex but in my case I don't even have to escape anything.